### ISSUE TITLE
Bank-APIs in 2025 evaluieren

### ISSUE THREAD

**Issue by Tosh79** [Tosh79/lamonee_backlog#134]

**Comment by rorar** [comment]
Alternative: https://beta-bridge.simplefin.org/

**Comment by Tactticus** [comment]
Wie ist hier der Stand der Dinge, Tobias? 

Ich danke dir, dass du es weiter auf der Liste hast und versuchst. 
Ist momentan wahrscheinlich noch unwirtschaftlich als kleiner Entwickler eine Schnittstelle zu erstellen, oder?

**Comment by Tosh79** [comment]
Ja, so sieht es aktuell leider immer noch aus. Die Anbindung einer Banking-Api würde erstmalig bedeuten, dass *lamonee einen gewissen Umsatz einfahren MUSS, weil ich den (aufs Jahre hochgerechnet locker mittel-4-stelligen) Fehlbetrag sonst aus meiner eigenen Tasche bezahlen müsste. Als Hobby-Projekt aktuell leider nicht tragbar.

Deshalb ist mein aktuelles Ziel, die Sichtbarkeit und den Nutzerstamm von *lamonee zu erhöhen (durch Internationalisierung und „native“ Apps in den App-Stores). Wenn der Plan aufgeht, wäre eine Banking-Api schon wieder denkbar.

Bis dahin soll *lamonee schlicht die beste Haushaltsbuch-App ohne Auto-Import sein. ;-)

**Comment by Tactticus** [comment]
Habe ich mir schon gedacht. 
Du bist da nicht unbedingt der einzige mit API-Einbindungsproblemen. Selbst, wenn man an eine Schnittsstelle kommt, hat man offenbar lange nicht die Möglichkeit sich in ALLE Institutionen einzubinden. Nicht mal alle aus Europa.... 

Selbst größere Firmen haben nicht mal alle Banken eingebunden, obwohl sie mehrere Schnittstellen zu Datenanbietern haben (finTS, finAPI via XS2A, banksAPI, Webscraping). Da fehlen dann teils sehr bekannte Institutionen wie **Norwegian Bank. Liechtensteinische Landesbank, Paysafe, Mintos, Interactive Brokers** oder etwas kleinere wie **Blue Wallet oder InRento**. 

Gibt es denn eventuell vereinzelt _lizenzfreie Schnittstellen_, die eine Option darstellen würden?

**Comment by rorar** [comment]
Aus Odoo kenne ich folgende 3rd party Konnektoren:
- Salt Edge
- Enablebanking
- Ponto (hier wird in Odoo einem eine 4€ Selbstzahleroption angeboten)
